src/app/index.html

```diff
@@ -84,6 +84,7 @@
 	<script src="lib/models/generic_collection.js"></script>
 	<script src="lib/models/movie_collection.js"></script>
 	<script src="lib/models/stream_info.js"></script>
+	<script src="lib/models/webtorrent_stream_info.js"></script>
```

Comment:
we should design this PR to nuke peerflix, can you be more specific about what you're missing ?